README.md

```diff
@@ -21,6 +23,28 @@ docker run -it -v /var/lib/kubelet/device-plugins:/var/lib/kubelet/device-plugin
 kubectl create -f nvidia-device-plugin.yml
 ```
 
+### Local Env
+
+You `MUST` set environment variable `LIBRARY_PATH` for static linking `cudart_static` library which required [here](https://github.com/NVIDIA/nvidia-docker/blob/master/src/cuda/bindings.go#L5), as show below (Ubuntu 16.04):
+
+**Note: please change `cuda-8.0` to your version**
+
+```shell
+export LIBRARY_PATH=/usr/local/cuda-8.0/lib64\
+${LIBRARY_PATH:+:${LIBRARY_PATH}}
+```
+
+#### Build Binary
+```shell
+$ go build
+```
+
+#### Run locally
+```shell
+$ ./k8s-device-plugin
```

Comment:
You will need `sudo`.
Also remove the `$`, to be consistent with the rest of the README.